app/helpers/carousel_helper.rb

```diff
@@ -0,0 +1,22 @@
+module CarouselHelper
+	def run_browser_ids(run_id)
+		browserids = Array.new
+		TestStatus.where(run_test_id: run_id).find_each do |ts|
+			browserids << ts.browser_type_id
+		end
+		return browserids
+	end
```

Comment:
Initially I had this function outputting a single browser ID but decided to use an array instead so that the carousel_playback_button method can just iterate over all of the browsers used when building the required HTML.

I prefer this way but that may well be incorrect. Worth discussing during the meeting today.